src/status_im/android/platform.cljs

```diff
@@ -37,17 +37,16 @@
                                          :border-top-width 0.5}}
    :discover              {:subtitle {:color     styles/color-gray2
```

Comment:
What do you think about putting discover styles in discover dir? Seems to be what we are doing with most other screens. Having it at top level seems a bit off imo, but I understand this is where it was initially.

Reply:
That's the place for platform specific styles only. The other namespace you mentioned is for platform agnostic ones. To be honest, I don't like it - files like this one are a hell to git manage, among other things. Maybe we could or should put platform conditioned styles in component specific stylesheets, but let's propose/do that in a separate refactoring task.

Reply:
How did you do the same in Wallet?

Reply:
Like this: https://github.com/status-im/status-react/blob/develop/src/status_im/ui/screens/wallet/main/styles.cljs#L65-L66

Agree re refactor task, let's skip for now